tests/functional/lang/eval-fail-mutual-recursion.nix

```diff
@@ -0,0 +1,32 @@
+# Check that "duplicate traces omitted" clears the set of "seen traces" after
+# eliding a group of traces.
```

Comment:
```suggestion
# Check that deduplication only affects consecutive intervals, and that they
# are reported independently of any preceding sections, even if they're
# indistinguishable.
#
# In terms of the current implementation, we check that we clear
# the set of "seen traces" after eliding a group of traces.
```

Trying to phrase it more as a requirement, rather than an implementation "detail".